instrumentation/grpc/spec/example/Gemfile

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source "https://rubygems.org"
+
+gem "opentelemetry-api"
+gem "opentelemetry-common"
+gem "opentelemetry-instrumentation-gruf", path: "../../gruf"
```

Comment:
It looks like this example may have been a carryover from the gruf instrumentation! Would you rather update it for the gRPC gem, or remove it and create an issue to add an example later on?

Reply:
I'll make an issue to add a gRPC-specific example here instead. :)